nodejs.spec.in

```diff
@@ -726,6 +730,13 @@ rm -f test/parallel/test-dns-cancel-reverse-lookup.js \
       test/parallel/test-dns-resolveany.js
 # multicast test fail since no socket?
 rm -f test/parallel/test-dgram-membership.js
+%if 0%{?fedora_version}
```

Comment:
on SLE these policies are disabled `OPENSSL_SYSTEM_CIPHERS_OVERRIDE=xyz_nonexistent_file`. I guess on Fedora these are outright removed from the library?

Reply:
test/parallel/test-crypto-certificate.js tests SPKAC which uses MD5 for verification.

Fedora has the following patch:

https://src.fedoraproject.org/rpms/openssl/blob/rawhide/f/0006-Disable-signature-verification-with-totally-unsafe-h.patch

It requires OPENSSL_ENABLE_MD5_VERIFY=1 to be set which should make the test pass I guess ...